AnkiDroid/src/main/java/com/ichi2/anki/CardTemplateEditor.java

```diff
@@ -704,6 +710,37 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
 
 
+        private boolean isFormatValid() {
+            boolean valid = true;
+            JSONObject template = getCurrentTemplate();
+
+            if (template != null && mTemplateEditor.getFieldNames() != null){
+                String tempAFMT = template.getString("afmt");
+                String tempQFMT = template.getString("qfmt");
+
+                List<String> fieldNames = new ArrayList<>(mTemplateEditor.getFieldNames());
+                fieldNames.add("FrontSide");
+                try{
+                    Matcher matcher = Pattern.compile("\\{\\{([^\\{\\}]*)\\}\\}").matcher(tempQFMT+tempAFMT);
```

Comment:
```suggestion
                    Matcher matcher = Pattern.compile("\\{\\{([^\\{\\}]*)\\}\\}").matcher(tempQFMT + tempAFMT);
```
i would suggest you to go through [code style](https://github.com/ankidroid/Anki-Android/wiki/Code-style)